bench/config.yaml

```diff
@@ -90,27 +75,87 @@ versions:
 #    parent: <tag/branch/commithash> # version to compare with in .diff graphs
 
 
-# - ghcide-v0.0.5
-# - ghcide-v0.0.6
-# - ghcide-v0.1.0
-# - ghcide-v0.2.0
-# - ghcide-v0.3.0
-# - ghcide-v0.4.0
-# - ghcide-v0.5.0
-# - ghcide-v0.6.0
-# - ghcide-v0.7.0
-# - ghcide-v0.7.1
-# - ghcide-v0.7.2
-# - ghcide-v0.7.3
-# - ghcide-v0.7.4
-# - ghcide-v0.7.5
-# - 1.0.0
-# - ghcide-v1.1.0
-# - ghcide-v1.2.0
-# - ghcide-v1.3.0
-- upstream: origin/master
+# - 1.8.0.0
+# - upstream: origin/master
+# - HEAD~1
 - HEAD
 
-# Heap profile interval in seconds (+RTS -i)
-# Comment out to disable heap profiling
-profileInterval: 1
+configurations:
+- None: []
+- Core:
+  - callHierarchy
+  - codeRange
+  - eval
+  - ghcide-code-actions-bindings
+  - ghcide-code-actions-fill-holes
+  - ghcide-code-actions-imports-exports
+  - ghcide-code-actions-type-signatures
+  - ghcide-completions
+  - ghcide-type-lenses
+  - pragmas
+- Ghcide:
+  - ghcide-code-actions-bindings
+  - ghcide-code-actions-fill-holes
+  - ghcide-code-actions-imports-exports
+  - ghcide-code-actions-type-signatures
+  - ghcide-completions
+  - ghcide-type-lenses
+- All:
+  - alternateNumberFormat
+  - callHierarchy
+  - changeTypeSignature
+  - class
+  - codeRange
+  - eval
+  - explicitFixity
+  - floskell
+  - fourmolu
+  - gadt
+  - ghcide-code-actions-bindings
+  - ghcide-code-actions-fill-holes
+  - ghcide-code-actions-imports-exports
+  - ghcide-code-actions-type-signatures
+  - ghcide-completions
+  - ghcide-type-lenses
+  - hlint
+  - importLens
+  - moduleName
+  - ormolu
+  - pragmas
+  - qualifyImportedNames
+  - refineImports
+  - rename
+  - stylish-haskell
+- alternateNumberFormat
```

Comment:
weird indentation? I'm confused about this tbh.

Reply:
I'll add comments